MetaMask/metamask-extension
# updated responsive props for border radius

This PR is to update responsive props for border-radius in Box

* Fixes #16491 

## Manual Testing Steps

`yarn test:unit:jest .ui/components/ui/box/box.test.js   `

- Go to the latest storybook build
- Search for Box in UI
- Check the stories, controls, and documentation

## Pre-Merge Checklist


### + If there are functional changes:
